Changelog.md

```diff
@@ -201,7 +201,7 @@ Move Authenication related code from ..Server.ApplicationServices to
   * native support for all platforms supporting netstandard 2
   * Run this code before creating first DomainContext, 
   ``` csharp
-  DomainContext.DomainClientFactory = new OpenRiaServices.DomainServices.Client.Web.SoapDomainClientFactory()
```

Comment:
Release notes for previous versions should not be changed

Reply:
I can remake the changes i two separate commits.

Reply:
Resolved